addon-resizer/nanny/apis/nannyconfig/v1alpha1/defaults.go

```diff
@@ -39,3 +39,18 @@ func SetDefaults_NannyConfiguration(obj *NannyConfiguration) {
 		obj.MemoryPerNode = "0"
```

Comment:
They doesn't seem to be reasonable defaults.

Reply:
I think it is reasonable, it makes it clear if one specifies only cpu per node and doesn't pass memoryPerNode. Any other default would be really confusing IMO.

Reply:
Sorry, I didn't see that actual "defaults" are elsewhere. Ignore this comment :-)